ceph/rados/core_workflows.py

```diff
@@ -2607,12 +2607,13 @@ def get_osd_uuid(self, osd_id):
         osd_fsid = self.run_ceph_command(cmd=cmd_osd_info)["uuid"]
         return osd_fsid
 
-    def get_orch_device_list(self, node=None):
+    def get_orch_device_list(self, node=None, refresh=False):
```

Comment:
I think the `refresh` flag doesn't need to be a conditional thing, we can just add the --refresh as to is to the base command.

Reply:
Modified the code.